public-api/src/main/java/org/pmiops/workbench/privateWorkbench/PrivateWorkbenchConfig.java

```diff
@@ -0,0 +1,35 @@
+package org.pmiops.workbench.privateWorkbench;
+
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.privateWorkbench.api.ProfileApi;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.http.HttpHeaders;
+import org.springframework.web.context.annotation.RequestScope;
+
+import javax.servlet.http.HttpServletRequest;
+
+@org.springframework.context.annotation.Configuration
+public class PrivateWorkbenchConfig {
+
+  private static final String END_USER_API_CLIENT = "endUserApiClient";
+
+  @Bean(name=END_USER_API_CLIENT)
+  @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
+  public ApiClient workbenchApiClient(WorkbenchConfig workbenchConfig) {
+    ApiClient apiClient = new ApiClient();
+    apiClient.setBasePath(workbenchConfig.server.apiBaseUrl);
+    apiClient.setDebugging(workbenchConfig.firecloud.debugEndpoints);
+    apiClient.setConnectTimeout(10000);
```

Comment:
Did this do anything? I thought 10s might have actually been the default anyways

Reply:
Not sure. I can leave it out and if there are a lot of issues, we can revisit later.